react/features/deeplinking/components/index.js

```diff
@@ -0,0 +1 @@
+export { default as DeeplinkingDesktopPage } from './DeeplinkingDesktopPage';
```

Comment:
I want to point out that you're setting the precedent for having deep linking be one word in the codebase. I'm okay with that as long as you're doing it intentionally.

Reply:
(Just checked and a few places already refer to it as "deep linking", two words. You should pick one way to be consistent.)

Reply:
Will change it to DeepLinking than.

Reply:
Changed!